### ISSUE TITLE
xterm --enable-sixel-graphics

### ISSUE THREAD

**Issue by ThomasDickey** [smasher164/arewesixelyet#18]
The [comment](https://github.com/smasher164/arewesixelyet/blob/7d6ba6aae89dc9dda969458cb3b8f838af7b8390/docs/index.html#L470) about xterm's configure script was almost two years out-of-date when it was added in a3d53920dfe857dcd94124953ec831ea74c023dc

>@[smasher164](https://github.com/smasher164/arewesixelyet/commits?author=smasher164)
smasher164 committed on Apr 19, 2022

See notes for xterm [patch #359](https://invisible-island.net/xterm/xterm.log.html#xterm_359)
>Patch #359 - 2020/08/17
>enable SIXEL feature by default.

The comment about running as a VT340 is also an issue (the default is VT420, for more commonly used features).  But you can have both (also from 2020, in [patch #357](https://invisible-island.net/xterm/xterm.log.html#xterm_357)):

>[decGraphicsID](https://invisible-island.net/xterm/manpage/xterm.html#VT100-Widget-Resources:decGraphicsID) (class DecGraphicsID)
>               Allows a way to combine the graphics feature from certain DEC
>               terminals (125, 240, 241, 330, 340 or 382) with other emulation
>               levels which did not provide the graphics feature.  As in
>              [decTerminalID](https://invisible-island.net/xterm/manpage/xterm.html#VT100-Widget-Resources:decTerminalID), leading non-digit characters are ignored, e.g.,
>               "vt340" and "340" are the same.

**Comment by smasher164** [comment]
Thanks for catching this! I can just rephrase it to "enabled by default since patch 359" then.
